cmd/krel/cmd/changelog.go

```diff
@@ -524,8 +524,8 @@ func commitChanges(repo *git.Repo, branch string, tag semver.Version) error {
 			return errors.Wrapf(err, "checking out release branch %s", branch)
 		}
 
-		// Remove all other changelog files if we’re on the first RC
-		if len(tag.Pre) > 1 && tag.Pre[0].String() == "rc" && tag.Pre[1].String() == "1" {
+		// Remove all other changelog files if we’re on the the first offical release
+		if tag.Patch == 0 && len(tag.Pre) == 0 {
```

Comment:
We probably have to adapt the tests as well:
https://github.com/kubernetes/release/blob/cba1d9c10a6df1bcf06ec1047d06f9d40a72940d/cmd/krel/cmd/changelog_test.go#L220-L223

Reply:
yep, i ran locally but looks like i need some extra setup because got some timeouts, then i open the PR to see here :)

Reply:
done